pkg/client/s3client.go

```diff
@@ -258,6 +302,7 @@ func (c *S3Client) UploadObject(fileName, objectName, bucketName string) error {
 	if err != nil {
 		return err
 	}
+	fmt.Println()
 	klog.Infof("Upload completed successfully in %f seconds to location %s\n", time.Since(startTime).Seconds(), result.Location)
```

Comment:
How about just adding `\n` in this line itself?

Reply:
When \n is added in the log, log header ("I0829 12:19:02.058216    9348 s3client.go:306]")is still in the same line where progress details are printed and only the log message is in the next line.


> Uploading: Total read(bytes):2160034197    progress:99%**I0829 12:19:02.058216    9348 s3client.go:306]** 
> Upload completed successfully in 507.212286 seconds to location <location>

Hence added fmt.Println() so that both the log header and log message are in the next line.